samples/cpp/benchmark_app/main.cpp

```diff
@@ -61,9 +61,12 @@ bool parse_and_check_command_line(int argc, char* argv[]) {
     if (FLAGS_api != "async" && FLAGS_api != "sync") {
         throw std::logic_error("Incorrect API. Please set -api option to `sync` or `async` value.");
     }
-    if (FLAGS_api == "sync" && FLAGS_nireq > FLAGS_niter) {
-        throw std::logic_error(
-            "Number of iterations should be greater than number of infer requests when using sync API.");
+    if (FLAGS_api == "sync") {
+        if (FLAGS_t == 0)
+            &&(FLAGS_nireq > FLAGS_niter) {
```

Comment:
```suggestion
        if ((FLAGS_t == 0) && (FLAGS_nireq > FLAGS_niter)) {
```

Reply:
Well, this is how I initially written that line, but the clang-format checker won't like it. This is the form obtained using clang-format and passing the clang-format checker job. So, I'll be forced to ignore your suggestion.